src/main.rs

```diff
@@ -93,10 +97,10 @@ struct Res {
 fn main() {
     // handle command line args
     let args = Args::parse();
-    real_main(args)
+    real_main(&mut io::stdout(), args)
 }
 
-fn real_main(args: Args) {
+fn real_main(stdout: &mut dyn io::Write, args: Args) {
```

Comment:
```suggestion
fn real_main<W: io::Write>(stdout: &mut W, args: Args) {
```
Doesn't really matter here, but runtime perf will be slightly better with generics instead of using dynamic dispatch (`dyn`)

Reply:
https://github.com/replit/nix-editor/pull/27